model/gamut_compress.ctl

```diff
@@ -2,17 +2,25 @@
 // <ACESuserName>ACES 1.3 Look - Gamut Compress</ACESuserName>
 
 //
-// "LMT" for compressing out of gamut scene-referred values into AP1.
-// 
-// Input and output are ACES2065-1.
+// Gamut compressor to bring out of gamut scene-referred values into AP1
```

Comment:
> Gamut compressor to bring out-of-gamut scene-referred values into AP1

Without the dashes, I read "bring out" first but we are doing the inverse here!

Reply:
Did we ever use "compressor" as a word? I have always referred to it as the "Gamut Compression Operator".

Reply:
Agree about the hyphens.

I've always thought of it as a "compressor", just like an audio compressor. I'm sure I've used the term verbally in meetings, but not sure it's in any documentation. We need to decide on terminology now we are moving away from calling it a "gamut mapper".